### PR TITLE
[Validator] Improve `zh_TW` translation

### PR DESCRIPTION
**Pull request by pan93412**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | N/A
| License       | MIT

The original translation appears machine-generated, containing some unusual characters (壹).

Additionally, the missing translation has been completed.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.zh_TW.xlf [hunk 1]
```diff
@@ -4,467 +4,467 @@
...
+                <target>這個數值包含目前限制級別不允許的字元。</target>
             </trans-unit>
             <trans-unit id="107">
                 <source>Using invisible characters is not allowed.</source>
                 <target>不允許使用隱形字元。</target>
             </trans-unit>
             <trans-unit id="108">
                 <source>Mixing numbers from different scripts is not allowed.</source>
-                <target>不允許混合來自不同語系的數字。</target>
+                <target>不允許混合來自不同字形的數字。</target>
```
Comment (lumynou5) [inline src/Symfony/Component/Validator/Resources/translations/validators.zh_TW.xlf]: I assume it means users cannot mix like "1一๑" (these are Arabic, Chinese, and Thai numerals respectively), and the "script" here should be "文字".

Comment (pan93412) [reply src/Symfony/Component/Validator/Resources/translations/validators.zh_TW.xlf:423]: Good idea. Let me fix it.

Comment (pan93412) [reply src/Symfony/Component/Validator/Resources/translations/validators.zh_TW.xlf]: Good idea. Let me fix it.

```suggestion
                <target>不允許混合來自不同文字的數字。</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf [hunk 2]
```diff
@@ -104,19 +104,19 @@
...
             </trans-unit>
             <trans-unit id="123">
                 <source>Please provide a valid phone number.</source>
-                <target>請提供有效的手機號碼。</target>
+                <target>請提供有效的電話號碼。</target>
             </trans-unit>
             <trans-unit id="124">
                 <source>The checkbox has an invalid value.</source>
-                <target>無效的選框值。</target>
+                <target>核取方塊有無效的值。</target>
```
Comment (pan93412) [inline src/Symfony/Component/Form/Resources/translations/validators.zh_TW.xlf]:
```suggestion
                <target>核取方塊上有無效的值。</target>
```

**Review by Crystal-RainSlide — APPROVED** [review]
Reviewed

**Review by lumynou5 — APPROVED** [review]
LGTM

**Comment by fabpot** [comment]
Thank you @pan93412.

**Pull request merged**